examples/README.md

```diff
@@ -204,6 +204,53 @@ fluentd:
           firstline: /\d{4}-\d{1,2}-\d{1,2}/
 ```
 
+## Filter out specific pods
```

Comment:
nit, this is for specific containers in a pod since `k8s.pod.name` would be the attribute to use otherwise: https://github.com/open-telemetry/opentelemetry-specification/blob/a25d5f03ab58ecf88c09f635df97d2328b5ba237/semantic_conventions/resource/k8s.yaml#L50

Reply:
Agreed, will fix